Failing tests building with openjdk 11

<!--- Provide a general summary of the issue in the Title above -->

## Expected Behavior
All tests should pass.
<!--- If you're describing a bug, tell us what should happen -->
<!--- If you're suggesting a change/improvement, tell us how it should work -->

## Current Behavior
<!--- If describing a bug, tell us what happens instead of the expected behavior -->
<!--- If suggesting a change/improvement, explain the difference from current behavior -->

`XmlRoutineFormatterTest#noCommentNoDefinitionLanguage(org.schemaspy.output.xml.dom.XmlRoutineFormatterTest)`

`XmlRoutineFormatterTest#full(org.schemaspy.output.xml.dom.XmlRoutineFormatterTest)`

`XmlRoutineFormatterTest#noComment(org.schemaspy.output.xml.dom.XmlRoutineFormatterTest)`

These tests fail with:

```
java.lang.AssertionError: 
Expecting empty but was:<[Expected child nodelist length '1' but was '3' - comparing <definition...> at /database[1]/routines[1]/routine[1]/definition[1] to <definition...> at /database[1]/routines[1]/routine[1]/definition[1] (DIFFERENT),
    Expected node type 'CDATA Section' but was 'Text' - comparing <definition ...><![CDATA[something < 10 else]]></definition> at /database[1]/routines[1]/routine[1]/definition[1]/text()[1] to <definition ...>
            </definition> at /database[1]/routines[1]/routine[1]/definition[1]/text()[1] (SIMILAR),
    Expected text value 'something < 10 else' but was '
            ' - comparing <definition ...><![CDATA[something < 10 else]]></definition> at /database[1]/routines[1]/routine[1]/definition[1]/text()[1] to <definition ...>
            </definition> at /database[1]/routines[1]/routine[1]/definition[1]/text()[1] (DIFFERENT),
    Expected child 'null' but was '#cdata-section' - comparing <NULL> to <definition ...><![CDATA[something < 10 else]]></definition> at /database[1]/routines[1]/routine[1]/definition[1]/text()[2] (DIFFERENT),
    Expected child 'null' but was '#text' - comparing <NULL> to <definition ...>
         </definition> at /database[1]/routines[1]/routine[1]/definition[1]/text()[3] (DIFFERENT)]>
	at org.schemaspy.output.xml.dom.XmlRoutineFormatterTest.noComment(XmlRoutineFormatterTest.java:137)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:252)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:141)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:112)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
```


## Possible Solution
<!--- Not obligatory, but suggest a fix/reason for the bug, -->
<!--- or ideas how to implement the addition or change -->
Add `.normalizeWhitespace()` to the `DiffBuilder` in `org.schemaspy.testing.XmlOutputDiff`

## Steps to Reproduce (for bugs)
<!--- Provide a link to a live example, or an unambiguous set of steps to -->
<!--- reproduce this bug. Include code to reproduce, if relevant -->
1. Set `javac` to jdk 11
2. `./mvnw test` 

## Your Environment
<!--- Include as many relevant details about the environment you experienced the bug in -->
* Version used:
* Operating System and version (e.g. Windows 10):
```
lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.1 LTS
Release:	18.04
Codename:	bionic
```

* Java version:
```
 java -version
openjdk version "10.0.2" 2018-07-17
OpenJDK Runtime Environment (build 10.0.2+13-Ubuntu-1ubuntu0.18.04.4)
OpenJDK 64-Bit Server VM (build 10.0.2+13-Ubuntu-1ubuntu0.18.04.4, mixed mode)

javac -version
javac 10.0.2
```

--- a/src/test/java/org/schemaspy/testing/XmlOutputDiff.java
+++ b/src/test/java/org/schemaspy/testing/XmlOutputDiff.java
@@ -30,6 +30,7 @@ public class XmlOutputDiff {
 
     public static Diff diffXmlOutput(Builder actually, Builder expected) {
         return DiffBuilder.compare(expected)
+                .normalizeWhitespace()
                 .withTest(actually)
                 .withDifferenceEvaluator(DifferenceEvaluators.chain(DifferenceEvaluators.Default, new IgnoreUsingXPath("/database[1]/@type")))
                 .build();

--- a/src/test/java/org/schemaspy/view/MustacheCompilerTest.java
+++ b/src/test/java/org/schemaspy/view/MustacheCompilerTest.java
@@ -18,7 +18,6 @@
  */
 package org.schemaspy.view;
 
-import com.sun.nio.file.ExtendedOpenOption;
 import org.assertj.core.api.SoftAssertions;
 import org.junit.BeforeClass;
 import org.junit.Test;

